pallets/ddc-clusters/src/migrations.rs

```diff
@@ -402,3 +405,151 @@ pub mod v2 {
 		}
 	}
 }
+
+pub mod v3 {
+
+	use ddc_primitives::{ClusterId, ClusterNodesCount, ClusterStatus, DdcEra};
+	use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade, weights::Weight};
+	use sp_runtime::Saturating;
+	use sp_std::collections::btree_map::BTreeMap;
+	#[cfg(feature = "try-runtime")]
+	use sp_std::vec::Vec;
+
+	use crate::{
+		cluster::{Cluster, ClusterProps},
+		Clusters, Config, Pallet, LOG_TARGET,
+	};
+
+	#[derive(Decode)]
+	pub struct OldCluster<AccountId> {
+		pub cluster_id: ClusterId,
+		pub manager_id: AccountId,
+		pub reserve_id: AccountId,
+		pub props: ClusterProps<AccountId>,
+		pub status: ClusterStatus,
+	}
+
+	impl<AccountId> OldCluster<AccountId> {
+		fn migrate_to_v3(self) -> Cluster<AccountId> {
+			Cluster {
+				cluster_id: self.cluster_id,
+				manager_id: self.manager_id,
+				reserve_id: self.reserve_id,
+				props: self.props,
+				status: self.status,
+				last_validated_era_id: DdcEra::default(),
+			}
+		}
+	}
+
+	pub type OldNodeStatus = bool;
+	pub struct MigrateToV3<T>(sp_std::marker::PhantomData<T>);
+	impl<T: Config> OnRuntimeUpgrade for MigrateToV3<T> {
+		fn on_runtime_upgrade() -> Weight {
+			let current_version = Pallet::<T>::current_storage_version();
+			let onchain_version = Pallet::<T>::on_chain_storage_version();
+			let mut weight = T::DbWeight::get().reads(1);
+
+			if onchain_version == 2 && current_version == 3 {
+				let mut nodes_count_by_cluster: BTreeMap<ClusterId, ClusterNodesCount> =
```

Comment:
I don't see usages of the `nodes_count_by_cluster` variable, I think we should remove it